Extensions/xkit_preferences.js

```diff
@@ -1006,7 +1006,7 @@ XKit.extensions.xkit_preferences = new Object({
 
 		if (obj.name.startsWith("xkit_") && XKit.tools.get_setting("xkit_show_internals","false") === "false") { return ""; }
 
-		var blacklisted_extensions = ["xkit_installer"];
+		var blacklisted_extensions = ["xkit_installer","old_stats"];
```

Comment:
@wolvan they're literally drop in replacements for eachother—this is
nothing more then pruning duplicate extensions from our library

On Sat, Nov 21, 2015 at 10:49 AM Wolvan <notifications@github.com> wrote:

> In Extensions/xkit_preferences.js
> <https://github.com/new-xkit/XKit/pull/739#discussion_r45547476>:
>
> > @@ -1006,7 +1006,7 @@ XKit.extensions.xkit_preferences = new Object({
> >
> >  		if (obj.name.startsWith("xkit_") && XKit.tools.get_setting("xkit_show_internals","false") === "false") { return ""; }
> >
> > -		var blacklisted_extensions = ["xkit_installer"];
> > +		var blacklisted_extensions = ["xkit_installer","old_stats"];
>
> Oh right, you are doing silent uninstall here. Why do we do this? We
> SHOULD tell the user that they get replaced. Silently doing stuff is no
> better than what staff does
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/new-xkit/XKit/pull/739/files#r45547476>.
>